### PR TITLE
RC Integrals, Derivatives Experiment

### PR DESCRIPTION
**Pull request by NamanRawat1**
Fixes issue #659  

Apk:
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1428483/app-debug.zip)

Screenshots for the change: 
screenshot_2017-10-31-00-46-21-037_org fossasia pslab

### CODE REVIEW

**Comment by NamanRawat1** [comment]
@asitava1998 review the PR

File: app/src/main/java/org/fossasia/pslab/experimentsetup/electricalexperiments/RCIntegralandderivativeexperiment.java [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+        dataSet2.setColor(Color.BLUE);
+        dataSet2.setDrawValues(false);
+        dataSet2.setDrawCircles(false);
+        dataSets.add(dataSet1);
+        dataSets.add(dataSet2);
+        outputChart.setData(new LineData(dataSets));
+        outputChart.invalidate();
+    }
+
+    private class CalcDataPoint extends AsyncTask<Float, Void, Void> {
```
Comment (NamanRawat1) [inline app/src/main/java/org/fossasia/pslab/experimentsetup/electricalexperiments/RCIntegralandderivativeexperiment.java]: AsynTask interface cannot take other than Float values @asitava1998

File: app/src/main/java/org/fossasia/pslab/experimentsetup/electricalexperiments/RCIntegralandderivativeexperiment.java [hunk 2]
```diff
@@ -0,0 +1,201 @@
...
+        LineData data = new LineData();
+        outputChart.setData(data);
+    }
+
+    private void startExperiment() {
+        Runnable runnable = new Runnable() {
+            @Override
+            public void run() {
+                scienceLab.setSine1(frequency);
+                for (float i = 0; i < 2000; i += 1) {
```
[the remark this answers is not in the document]
Comment (asitava1998) [reply app/src/main/java/org/fossasia/pslab/experimentsetup/electricalexperiments/RCIntegralandderivativeexperiment.java]: @NamanRawat1  You can use int here. However while sending the data to the Async Task you can typecast it to float.

**Comment by NamanRawat1** [comment]
Updated apk @asitava1998 
[app-debug.zip](https://github.com/fossasia/pslab-android/files/1464397/app-debug.zip)

**Comment by CloudyPadmal** [comment]
Hello @NamanRawat1 ! Could you please reopen this PR? Travis build seems stopped! So that it will restart!

**Pull request merged**